# Entrega 4

- Fecha: 2021-11-16
- Prácticas: 7, 8

## Ejercicio 1

Considere la siguiente muestra aleatoria de tamaño 10 generada con función de densidad:

$f(x) = \tfrac{2 \beta}{\pi} x e^{\tfrac{-\beta x^2}{\pi}} I_{[0, +\infty)}(x)$

Donde $\beta > 0$ es un parámetro desconocido y que deseamos estimar. Además, $E(X) = \tfrac{\pi}{2 \sqrt{\beta}}$, $V(X) = \tfrac{\pi(4 - \pi)}{4 \beta}$

In [35]:
X = c(1.48, 1.67, 1.29, 1.11, 0.82, 0.27, 0.71, 1.48, 0.88, 1.79)

### Pregunta A

Estimar la $P(X \leq 1.2)$ usando la función de distribución empírica.

In [30]:
sum(X <= 1.2) / length(X)

[1] 0.5

### Pregunta B

Usando las observaciones dar una estimación de $\tfrac{1}{\beta}$ basada en el estimador de máxima verosimilitud (MV) de $\tfrac{1}{\beta}$.

$
L(\beta)
= \prod_{i=1}^n f(x_i)
= \prod_{i=1}^n \tfrac{2 \beta}{\pi} x_i e^{\tfrac{-\beta x_i^2}{\pi}}
= \Big( \tfrac{2 \beta}{\pi} \Big)^n \Big( \prod_{i=1}^n x_i \Big) \Big( e^{-\tfrac{\beta}{\pi} \sum_{i=1}^n x_i^2} \Big)
$

$
ln(L(\beta))
= n ln(\tfrac{2\beta}{\pi}) + \sum_{i=1}^n ln(x_i) - \tfrac{\beta}{\pi} \sum_{i=1}^n x_i^2
= n ln(\tfrac{2\beta}{\pi}) + \sum_{i=1}^n ln(x_i) - \tfrac{\beta n \overline{X^2}}{\pi}
$

$
\tfrac{\partial}{\partial \beta} ln(L(\beta))
= \tfrac{n}{\tfrac{2\beta}{\pi}} \tfrac{2}{\pi} - \tfrac{n \overline{X^2}}{\pi}
= \tfrac{n}{\beta} - \tfrac{n \overline{X^2}}{\pi}
$

$
\tfrac{\partial}{\partial \beta} ln(L(\beta)) = 0
\iff \tfrac{n}{\beta} - \tfrac{n \overline{X^2}}{\pi} = 0
\iff \tfrac{n}{\beta} = \tfrac{n \overline{X^2}}{\pi}
\iff \hat{\beta} = \tfrac{\pi}{\overline{X^2}}
\iff \tfrac{1}{\hat{\beta}} = \tfrac{\overline{X^2}}{\pi}
$

In [38]:
b_MV = pi / mean(X^2)
round(1 / b_MV, 4)

[1] 0.4868

### Pregunta C

Dar una estimación de $P(X \leq 1.2)$ basada en el estimador calculado arriba.

In [39]:
f = function(x) (2 * b_MV) / pi * x * exp(-b_MV * x^2 / pi)
round(integrate(f, 0, 1.2)$value, 4)

[1] 0.61

### Pregunta D

El estimador de $\tfrac{1}{\beta}$, ¿es consistente?

$
E(X^2)
= V(X) + E(X)^2
= \tfrac{\pi(4 - \pi)}{4 \beta} + \big( \tfrac{\pi}{2 \sqrt{\beta}} \big)^2
= \tfrac{4 \pi - \pi^2}{4 \beta} + \tfrac{\pi^2}{4 \beta}
= \tfrac{\pi}{\beta}
$

Por la LGN: $\tfrac{1}{\hat{\beta}} = \tfrac{\overline{X^2}}{\pi} \xrightarrow{n \rightarrow \infty} \tfrac{E(X^2)}{\pi} = \tfrac{1}{\pi} \tfrac{\pi}{\beta} = \tfrac{1}{\beta}$

$
P(|\tfrac{1}{\hat{\beta}} - \tfrac{1}{\beta}| > \epsilon)
\xrightarrow{n \rightarrow \infty} P(|\tfrac{1}{\beta} - \tfrac{1}{\beta}| > \epsilon) = P(0 > \epsilon) = 0 \; \forall \epsilon > 0
$

$\implies \tfrac{1}{\hat{\beta}}$ es consistente.

### Pregunta E

El estimador de $\tfrac{1}{\beta}$, ¿es insesgado?

$
E \Big( \tfrac{1}{\hat{\beta}} \Big)
= E \Big( \tfrac{\overline{X^2}}{\pi} \Big)
= \tfrac{1}{\pi} E(\overline{X^2})
= \tfrac{1}{\pi} E(X^2)
= \tfrac{1}{\pi} \tfrac{\pi }{\beta}
= \tfrac{1}{\beta}
$

$\implies \tfrac{1}{\hat{\beta}}$ es insesgado.

## Ejercicio 2

Se extrae una muestra aleatoria de una población $\mathcal{N}(\mu, \sigma^2)$ obteniéndose para estas 22 observaciones una media muestra de $4.59$ y varianza muestral de $3.46$.

Construir un intervalo de confianza de nivel exacto $98\%$ para la varianza de esta distribución.

Usar la convención de dejar igual cantidad de área a cada lado bajo la curva de la distribución pivote elegida.

### Pregunta A

Indique qué distribución usará para construir el intervalo (Normal, t-Student, Chi cuadrado, Gamma, otra).

**Chi cuadrado**

$\Big[ \frac{(n-1)S^2}{\mathcal{X}^2_{n-1,1-\alpha/2}}, \frac{(n-1)S^2}{\mathcal{X}^2_{n-1,\alpha/2}} \Big]$

In [48]:
alpha = 0.02
n = 22
var = 3.46
round(c((n - 1) * var / qchisq(1 - alpha/2, n - 1), (n - 1) * var / qchisq(alpha/2, n - 1)), 4)

[1] 1.8663 8.1666

### Pregunta B

Con la información obtenida, el extremo izquierdo del intervalo estimado es: **1.8663**

### Pregunta C

Con la información obtenida, el extremo derecho del intervalo estimado es: **8.1666**

### Pregunta D

¿Puede asegurar que hay una probabilidad de $0.98$ de que si se observa un nuevo dato de esta población, este pertenezca al intervalo construido?

**No**

### Pregunta E

¿Puede asegurar que el $98\%$ de las observaciones pertenece al intervalo construido?

**No**